File: Plugins/Aspose_Words_NET_for_PHP/src/aspose/words/quickstart/HelloWorld.php
<?php
/**
 * Created by PhpStorm.
 * User: assadmahmood
 * Date: 30/06/15
 * Time: 10:45 AM
 */

namespace Aspose\Words\QuickStart;


class HelloWorld {

    public static function run($dataDir=null)
    {
        if(is_null($dataDir)) die("Data Directory Undefined");

        $doc = new \COM("Aspose.Words.Document");

        $builder = new \COM("Aspose.Words.DocumentBuilder");

        $builder->Document = $doc;

        $builder->Write("Hello world!");

        $doc->Save($dataDir . "/HelloWorld Out.docx");
    }

} 

File: Examples/DocsExamples/DocsExamples/Getting started/Hello world.cs
using Aspose.Words;
using NUnit.Framework;

namespace DocsExamples.Getting_started
{
    public class Hello_world : DocsExamplesBase
    {
        [Test]
        public void HelloWorld()
        {
            //ExStart:HelloWorld
            //GistDesc:Getting started example
            Document docA = new Document();            
            DocumentBuilder builder = new DocumentBuilder(docA);

            // Insert text to the document start.
            builder.MoveToDocumentStart();
            builder.Write("First Hello World paragraph");

            Document docB = new Document(MyDir + "Document.docx");
            // Add document B to the and of document A, preserving document B formatting.
            docA.AppendDocument(docB, ImportFormatMode.KeepSourceFormatting);
            
            docA.Save(ArtifactsDir + "C:\\Temp\\output_AB.pdf");
            //ExEnd:HelloWorld
        }
    }
}
